Monopoly/Assets/Game.cs

```diff
@@ -96,6 +97,25 @@ void Update()
 
             if (players[currentPlayer].PawnMoved())
             {
+                int currentPlayerPosition = players[currentPlayer].GetCurrentPosition();
+                int currentPlayerId = players[currentPlayer].GetId();
+                Property currentPlayerStandingProperty = properties[currentPlayerPosition];
+
+                if (currentPlayerStandingProperty.HasOwner())
+                {
+                    HandleRentPay(currentPlayerStandingProperty, currentPlayerId);
+                }
+                else
+                {
+                    if (currentPlayerStandingPropert.IsOwningBy(currentPlayerId))
```

Comment:
typo: PropertY

Reply:
Aha, tutaj sprawdzamy czy pole jest własnością aktywnego gracza, przeniósłbym to wcześniej, bo teraz chyba ten kod jest unreachable? chyba że hasOwner zwraca tylko true dla ownerów innych niż aktywny gracz, ale.. chyba nie i nie powinno tak być :D

Reply:
jeszcze ta literówka i git: currentPlayerStandingPropertY